src/sessions/__tests__/VirtualPageLoadTimer.test.ts

```diff
@@ -0,0 +1,567 @@
+import { advanceTo } from 'jest-date-mock';
+import { PageManager } from '../PageManager';
+import {
+    DEFAULT_CONFIG,
+    mockFetch,
+    mockFetchWith500,
+    mockFetchWithError,
+    mockFetchWithErrorObject,
+    mockFetchWithErrorObjectAndStack
+} from '../../test-utils/test-utils';
+import { Config } from '../../orchestration/Orchestration';
+import { PERFORMANCE_NAVIGATION_EVENT_TYPE } from '../../plugins/utils/constant';
+import { PAGE_VIEW_EVENT_TYPE } from '../../plugins/utils/constant';
+import mock from 'xhr-mock';
+
+const record = jest.fn();
+
+declare const jsdom: any;
+
+Object.defineProperty(document, 'referrer', {
+    value: 'https://console.aws.amazon.com'
+});
+Object.defineProperty(document, 'title', { value: 'Amazon AWS Console' });
+global.fetch = mockFetch;
+
+describe('PageManager tests', () => {
+    let url;
+
+    beforeAll(() => {
+        url = window.location.toString();
+    });
+
+    beforeEach(() => {
+        advanceTo(0);
+        mock.setup();
+        record.mockClear();
+        jsdom.reconfigure({
+            url: url
+        });
+    });
+
+    afterEach(() => {
+        jest.resetModules();
+    });
+
+    test('when next page is created then items in requestBuffer should be added to ongoingRequests', async () => {
+        // Init
+        const config: Config = {
+            ...DEFAULT_CONFIG,
+            allowCookies: true
+        };
+        const pageManager: PageManager = new PageManager(config, record);
```

Comment:
issue: We should be instantiating VirtualPageLoadTimer directly in this test suite (i.e., we should not instantiate PageManager), since we are testing VirtualPageLoadTimer. 

There is also only one function we need to test -- `startTiming`